tests/test/test.js

```diff
@@ -642,6 +642,34 @@ describe('Exporter', function() {
                 };
                 assert.deepStrictEqual(customNormalHash, expectedCustomNormalHash);
             });
+
+            it('exports unlit materials', function() {
```

Comment:
This test needs to be protected by `if (blenderVersion !== 'blender279b')`, similar to [line 745](https://github.com/KhronosGroup/glTF-Blender-IO/blob/59f59b721ecdb278730c41f1a7cd7933c53b11b6/tests/test/test.js#L745) for example.